src/main/java/seedu/foodrem/model/item/ItemExpiryDate.java

```diff
@@ -74,12 +72,4 @@ public int hashCode() {
     public String toString() {
         return expiryDate == null ? "" : expiryDate.format(EXPIRY_DATE_FORMATTER);
     }
-
```

Comment:
May I ask why `toListView()` has been removed from all the Item fields?

Reply:
The responsibility for showing the UI is not with item and should be with item card. 

In future if there is a new UI component we should let that component decide how to display item.

If we continue letting item control how it is displayed in different components, Item will become a godClass for Ui.

This violated SRP.

Reply:
I understand. Referring to [this link](https://code.tutsplus.com/tutorials/solid-part-1-the-single-responsibility-principle--net-36074), can we then move this formatting within the item card to a method? Just for reusability

Reply:
Yixian: Will be done after Richard's UI changes